hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -2976,6 +2976,22 @@ public boolean supportsNoWait() {
 		return false;
 	}
 
+	/**
+	 * Does this dialect/database support LO_CREATE and related functions.
+	 *
+	 * @return {@code true} if LO_CREATE and related functions are supported
+	 */
+	public boolean supportsLoFunctions() {
+		return false;
+	}
+
+	/**
+	 * Does this dialect/database support mixed type arithmetic (e.g. <int> + <float>).
```

Comment:
Given '<' and '>' are special HTML chars, I saw only "(e.g. +)" shows up in javadoc. You might need html escaped text for '<' and '>'.

Reply:
I removed the '<' and '>' characters since I don't think they are needed.